tests/generic_config_updater/test_pfcwd_interval.py

```diff
@@ -120,13 +120,49 @@ def get_new_interval(duthost, is_valid):
         return min(detection_time, restoration_time) + 10
 
 
+def test_stop_pfcwd(duthost):
+    start_pfcwd = duthost.shell('config pfcwd start_default')
+    pytest_assert(not start_pfcwd['rc'], "Failed to start default pfcwd config")
+    pfcwd_config = duthost.shell("show pfcwd config")
+    pytest_assert(not pfcwd_config['rc'], "Unable to read pfcwd config")
+
+    for line in pfcwd_config['stdout_lines']:
+        if line.startswith('Ethernet'):
```

Comment:
>line [](http://example.com/codeflow?start=11&length=4)

If there is no `Ethernet` at all, later code will throw.

Reply:
Thanks for the note, I switched to for-else to throw if `interface` is not defined